src/tools/clippy/tests/ui-toml/absolute_paths/absolute_paths.allow_crates.stderr

```diff
@@ -5,6 +5,7 @@ LL |     std::f32::MAX;
    |     ^^^^^^^^^^^^^
    |
    = note: `-D clippy::absolute-paths` implied by `-D warnings`
+   = help: add `#[allow(clippy::absolute_paths)]` to override `-D warnings`
```

Comment:
```suggestion
   = help: fix this {error|warning} or add `#[allow(clippy::absolute_paths)]` to override `-D warnings`
```
Maybe amend the wording to highlight that adding `allow` is usually not the preferred way to address lints?

Reply:
Why not, but in that case wouldn't it be better to inverse the sentences?
```diff
-   = help: add `#[allow(clippy::absolute_paths)]` to override `-D warnings`
+   = help: to override `-D warnings` add `#[allow(clippy::absolute_paths)]`
```
This would make it clearer that it should only be done to override the flag.

Reply:
Seems fine too.